cmd/instance_pool_create.go

```diff
@@ -33,13 +32,14 @@ type instancePoolCreateCmd struct {
 	InstancePrefix     string            `cli-usage:"string to prefix managed Compute instances names with"`
 	InstanceType       string            `cli-usage:"managed Compute instances type (format: [FAMILY.]SIZE)"`
 	Labels             map[string]string `cli-flag:"label" cli-usage:"Instance Pool label (format: key=value)"`
+	MinAvailable       int64             `cli-usage:"Minimum number of running Instances"`
 	PrivateNetworks    []string          `cli-flag:"private-network" cli-usage:"managed Compute instances Private Network NAME|ID (can be specified multiple times)"`
-	SSHKey             string            `cli-flag:"ssh-key" cli-usage:"SSH key to deploy on managed Compute instances"`
+	SSHKeys            []string          `cli-flag:"ssh-key" cli-usage:"SSH key to deploy on managed Compute instances (can be specified multiple times)"`
```

Comment:
Can you explain how this change is relevant to this PR?

Reply:
I made it based on similar migration to v3 made by @pierre-emmanuelJ  this was my reference: https://github.com/exoscale/cli/commit/cc09480f405a7cf663caa5588be20b9cc8bc4296#:~:text=SSHKeys%20%20%20%20%20%20%20%20%20%20%20%20%5B%5Dstring%20%20%20%20%20%20%20%20%20%20%60cli%2Dflag%3A%22ssh%2Dkey%22%20cli%2Dusage%3A%22SSH%20key%20to%20deploy%20on%20the%20instance%20(can%20be%20specified%20multiple%20times)%22%60

Reply:
See https://github.com/exoscale/cli/pull/629#discussion_r1723571354